runtimes/pythonrt/py_test.go

```diff
@@ -17,10 +17,14 @@ import (
 )
 
 func skipIfNoPython(t *testing.T) {
-	_, err := exec.LookPath("python")
-	if err != nil {
+	_, err := pyExeInfo(context.Background())
```

Comment:
could you use the same func in test_createVEnv as well?

Reply:
Done